web/preferences.js

```diff
@@ -113,6 +119,9 @@ class BasePreferences {
       return; // Invalid preference.
     }
     AppOptions.set(name, value);
+    if (this.eventBus && BasePreferences.#eventToDispatch.has(name)) {
+      this.eventBus.dispatch(name.toLowerCase(), { source: this, value });
+    }
```

Comment:
This can be shortened slightly, and please let's add a blank line.
```suggestion

    if (BasePreferences.#eventToDispatch.has(name)) {
      this.eventBus?.dispatch(name.toLowerCase(), { source: this, value });
    }
```